src/FSAL/FSAL_CORTXFS/handle.c

```diff
@@ -1486,16 +1488,18 @@ static  inline fsal_status_t kvsfs_getattrs(struct fsal_obj_handle *obj_hdl,
 
 	myself =
 		container_of(obj_hdl, struct kvsfs_fsal_obj_handle, obj_handle);
+	retval = cfs_fh_from_ino(myself->cfs_fs, &ino, &fh);
+	stat = cfs_fh_stat(fh);
 
-	retval = cfs_getattr(myself->cfs_fs, &cred,
-			     kvsfs_fh_to_ino(myself->handle), &stat);
+	//retval = cfs_getattr(myself->cfs_fs, &cred,
```

Comment:
After this point, we want be needing `cfs_getattr` so it is good to remove this commented part.